app.controllers.js

```diff
@@ -53,3 +53,13 @@ exports.getCommentsByArticleId = (req, res, next) => {
     next(err)
   })
 }
+
+exports.postComment = (req, res, next) => {
+  const { article_id } = req.params;
+  const { username, body } = req.body;
+    return postCommentOnArticle(username, body, article_id)
+    .then((postedComment) => {
+      res.status(201).send({msg: "comment posted!", comment: postedComment[0].body})
```

Comment:
We should always be sending an object with a key containing the value of what we want to respond with. In this case we should only have a single key of comment with the value of the whole posted comment object